crates/editor_tabs/src/start_layout.rs

```diff
@@ -0,0 +1,108 @@
+use crate::{
+    tab_name::{TabName, TabNameHolder},
+    EditorUi,
+};
+/// This mod contains start layouts templates for tabs
+/// It was created for easy customization of shown tabs on startup
+use bevy::prelude::*;
+
+pub trait StartLayout {
+    fn build(&self) -> egui_dock::DockState<TabNameHolder>;
+}
+
+pub trait GroupLayout<G>: StartLayout {
+    /// Add tab to group to end
+    fn push<N: TabName>(&mut self, group: G, tab: N);
+    /// Add tab to group to start
+    fn push_front<N: TabName>(&mut self, group: G, tab: N);
+}
+
+pub enum DoublePanel {
+    TopPanel,
```

Comment:
Maybe call them TopLeftPanel and BottomLeftPanel? first time I read them I felt confused with the menutoolbars

Reply:
Maybe then call TopLeft and BottomLeft?  I love reducing words count